CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.7.0 (TBD)
+
+### New Features
+
+- Added parameters `external_access_integrations` and `secrets` when creating a UDF, UDTF or Stored Procedure from Snowpark Python to allow integration with external access.
```

Comment:
Will UDAF have this feature?

Reply:
yes, udaf is expected support this feature, but udaf needs more tests from backend and it's lacking secrets support, so more work expected from UCLE work: https://snowflakecomputing.atlassian.net/browse/SNOW-886334.

I created a JIRA for the future client work: https://snowflakecomputing.atlassian.net/browse/SNOW-886368.

I prefer pushing the udaf client change until the UCLE team confirms the backend work is done